pages/models/regulations/governor/GovSteamEU/index.md

```diff
@@ -0,0 +1,443 @@
+definitions of initial states below.
+
+<span id="eq-initReheat">$$
+R_\mathrm{h\,0} = 
+\begin{cases}
+    P_\mathrm{m\,0} ,& \text{if } T_\mathrm{rh} > 0\\
+    0,              & \text{otherwise}
+\end{cases}
+ \qquad(1)$$</span>
+
+<span id="eq-initPc">$$
+P_\mathrm{c\,0} = P_{\text{m}\,0} - \min\left(\max\left(K_{\omega_{\text{cor}}} \cdot 1, \omega_{\text{min}}\right), \omega_{\text{max}}\right)
+ \qquad(2)$$</span>
+
+### Initial states
+
+These initial states refer to the states as defined in
+<a href="#fig-control_diagram" class="quarto-xref">Figure 1</a>.
+
+<span id="eq-initXlp">$$
+x_\mathrm{lp\,0} = P_\mathrm{m\,0}
+ \qquad(3)$$</span>
+
+<span id="eq-initXrh">$$
+x_\mathrm{rh\,0} = R_\mathrm{h\,0}
+ \qquad(4)$$</span>
+
+<span id="eq-initXhp">$$
+x_\mathrm{hp\,0} = P_\mathrm{m\,0}
+ \qquad(5)$$</span>
+
+<span id="eq-initXb">$$
+x_\mathrm{b\,0} = 1
+ \qquad(6)$$</span>
+
+<span id="eq-initXhhp">$$
+x_\mathrm{hhp\,0} = P_\mathrm{m\,0}/x_\mathrm{b}
+ \qquad(7)$$</span>
+
+<span id="eq-initXvip">$$
+x_\mathrm{vlp\,0} = 1
+ \qquad(8)$$</span>
+
+<span id="eq-initXomega">$$
+x_\mathrm{\omega\,0} =  \omega_\mathrm{0}
+ \qquad(9)$$</span>
+
+<span id="eq-initXen">$$
+x_\mathrm{en\,0} = P_\mathrm{c\,0}
+ \qquad(10)$$</span>
+
+<span id="eq-initXip">$$
+x_\mathrm{ip\,0} = P_\mathrm{c\,0}
+ \qquad(11)$$</span>
+
+<span id="eq-initXdp">$$
+x_\mathrm{dp\,0} = 0
+ \qquad(12)$$</span>
+
+<span id="eq-initXp">$$
+x_\mathrm{p\,0} = P_\mathrm{m\,0}
+ \qquad(13)$$</span>
+
+<span id="eq-initXf">$$
+x_\mathrm{f\,0} = f_\mathrm{0}
+ \qquad(14)$$</span>
+
+## Open Questions?
+
+- The island mode control signal does not pass thourgh the electro
+  hydraulic transducer. Why?
+- why are the multiplication blocks that model the valve position added
+  **inside** the first order lags (boiler, reheater) and not outside
+  (e.g. just after it)? (it leads to different dynamic behaviour)
+- Are there cases where it makes sense to use grid connected ($$f$$) and
+  island ($$\omega$$) control mode simultaneously or should one of them
+  always be deactivated?
+
+## Open source implementations
+
+This model has been successfully implemented in :
+
+| Software               | URL                                        | Language | Open-Source License                                | Last consulted date | Comments                                                                                              |
+| ---------------------- | ------------------------------------------ | -------- | -------------------------------------------------- | ------------------- | ----------------------------------------------------------------------------------------------------- |
+| Open Modelica / Dynawo | [Dynawo](https://github.com/dynawo/dynawo) | modelica | [MPL v2.0](https://www.mozilla.org/en-US/MPL/2.0/) | 16/07/2024          | For modeling assumptions and test results, see [Dynawo](https://github.com/dynawo/dynawo) repository. |
```

Comment:
Is there a Dynawo model standing for GovSteamEU ? There is a GovSteam1 model, which differs from GovSteamEU.

Reply:
Yes, it is under development and soon to be merged. 
https://github.com/dynawo/dynawo/issues/3112